### PR TITLE
Pytorch Object Detection and Tracking using Darknet

### PR DESCRIPTION
**Pull request by somi-agarwal**

### CODE REVIEW

**Comment by Toms42** [comment]
Are you planning on merging this soon? @somi-agarwal

File: tauv_common/src/vision/config/robosub.names [hunk 1]
```diff
@@ -0,0 +1,4 @@
+dice1
```
Comment (Toms42) [inline tauv_common/src/vision/config/robosub.names]: ideally these config things should be moved into the tauv_config folder, but I think this is ok for now.

**Comment by Toms42** [comment]
Also, are you planning on writing a ROS node wrapper for this? Basically to publish a list of bounding boxes and setup the video pipeline? That would be ideal.

**Comment by sagarwal-atg** [comment]
> Also, are you planning on writing a ROS node wrapper for this? Basically to publish a list of bounding boxes and setup the video pipeline? That would be ideal.

Yes, its a WIP.

**Pull request closed without merging**